CHANGES.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 6.2 (unreleased)
 ----------------
 
+- Avoid error in search when the parameter received by ``IndexQuery`` is a ``record``.
```

Comment:
You should add a reference to the original issue (https://github.com/plone/Products.CMFPlone/issues/3007) using the notation you see for other issues in this change log further down.

Reply:
Done.